drivers/lock.py

```diff
@@ -23,6 +23,8 @@
 
 VERBOSE = True
 
+LOCK_TYPE_GC_RUNNING = "gc_running"
```

Comment:
Do we risk having a clash with a pre-existing runing process during an upgrade where the running GC process already has the `running` lock and then a new process tries, and succeeds, in acquiring the `gc_running` lock and assumes it has free reign over the GC process? That probably results in data corruption unless we ensure the upgrade aborts any in progress GC operations, which it does not by default do.